### ISSUE TITLE
it looks success but should be fail when puppeteer requires different chromium revision

### ISSUE THREAD

**Issue by pastak** [hitode909/backstop-retry-failed-scenarios#75]
Set environment value `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` then puppeteer skips downloading chromium and chromium revision's validation checking. puppeteer requires different the revision from local chromium will fail but [`cli.ts`'s `main()`](https://github.com/hitode909/backstop-retry-failed-scenarios/blob/b42bfb0afc38bff90db329ff5e45cbb672c3e7f7/src/cli.ts#L65-L76) is async function so it will become unhandled rejection and will be ignored by node( < v15 or without `--unhandled-rejections=strict` ), node will exit with code 0 (success).

[![Image from Gyazo](https://i.gyazo.com/1fd7609cdec6ca71183092e92bee1b2a.png)](https://gyazo.com/1fd7609cdec6ca71183092e92bee1b2a)

**Comment by hitode909** [comment]
@pastak Thank you for reporting! Do you think this diff is enough?

```diff
+ try {
  await runner.run();
+ catch (error) { process.exit(1) };
```

**Comment by pastak** [comment]
@hitode909 It maybe enough to fix 👍

**Comment by hitode909** [comment]
@pastak Fixed in #82. Please use 1.2.3.
- https://www.npmjs.com/package/backstop-retry-failed-scenarios/v/1.2.3
